sw/device/silicon_creator/rom_ext/BUILD

```diff
@@ -125,6 +125,7 @@ cc_library(
         "//sw/device/silicon_creator/lib/base:chip",
         "//sw/device/silicon_creator/lib/base:sec_mmio",
         "//sw/device/silicon_creator/lib/base:static_critical_boot_measurements",
+        "//sw/device/silicon_creator/lib/base:static_critical_in_memory_epmp",
```

Comment:
I wasn't able to find this file. Is it missing from this commit snapshot?

Reply:
It was renamed to `static_critical_epmp_state`, fixed!